internal/updater/checker.go

```diff
@@ -62,43 +72,57 @@ func (u *Checker) CheckFor(desiredChannel, desiredVersion string) (*AvailableUpd
 	return info, nil
 }
 
-func (u *Checker) GetUpdateInfo(desiredChannel, desiredVersion string) (*AvailableUpdate, error) {
-	platform := runtime.GOOS + "-" + runtime.GOARCH
+func (u *Checker) infoURL(cfg Configurable, desiredVersion, branchName, platform string) string {
+	v := make(url.Values)
+	v.Set("channel", branchName)
+	v.Set("platform", platform)
+	v.Set("source", "update")
+
+	if desiredVersion != "" {
+		v.Set("target-version", desiredVersion)
+	}
+
+	tag := cfg.GetString(CfgUpdateTag)
+	if tag != "" {
+		v.Set("tag", tag)
+	}
```

Comment:
Tag should be passed in directly, rather than passing cfg. 

If it really needs to use cfg in the update checker then I'd rather we pass it to the struct rather than this specific call.

Reply:
Oh, now I remember why I did that: The `DefaultChecker` is initialized as a package variable.  So, we cannot easily pass arguments to it: https://github.com/ActiveState/cli/blob/1dcea3dcee8042cfbb9bf0ee22812038a5370fa6/internal/updater/checker.go#L34

Reply:
I should probably get rid of that DefaultChecker as a variable completely.  It is not saving us anything actually, as it is just returning the struct...

Reply:
Yeah makes sense, let's just drop it.